app/models/ipeds_ic_ay.rb

```diff
@@ -2,6 +2,8 @@
 class IpedsIcAy < ActiveRecord::Base
   include Loadable, Exportable
 
+  USE_COLUMNS = [:tuition_in_state, :tuition_out_of_state, :books].freeze
```

Comment:
I guess my biggest problem with this is USE_COLUMNS and MAP are less than ideal names for the constants. But if you are going to keep them outside of a single constants.rb, then it makes sense to have generic enough that you can use consistently across the various models.